tests/test_fgcmcal_hsc.py

```diff
@@ -51,6 +51,10 @@ def setUpClass(cls):
             cls.dataDir = lsst.utils.getPackageDir('testdata_jointcal')
         except LookupError:
             raise unittest.SkipTest("testdata_jointcal not setup")
+        try:
+            lsst.utils.getPackageDir('obs_subaru')
+        except lsst.pex.exceptions.NotFoundError:
```

Comment:
I think this can be a `LookupError` as above.

Reply:
Yes. We really don't want pex_exceptions in python code unless we have no choice.

Reply:
So I think this applies to the PRs on `jointcal` and `testdata_jointcal` as well. (And I believe @timj is who taught me to replace `lsst.pex.exceptions.NotFoundError` with `LookupError`).